tests/integration/test_enrollment/test_agent_auth_enrollment.py

```diff
@@ -24,31 +24,66 @@
 
 import pytest
 import os
+import socket
 
 from wazuh_testing.tools import LOG_FILE_PATH
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.services import control_service
 from wazuh_testing.tools.file import load_tests, truncate_file
-from wazuh_testing.tools.monitoring import FileMonitor, make_callback
+from wazuh_testing.tools.monitoring import FileMonitor, QueueMonitor, make_callback
+from wazuh_testing.agent import AgentAuthParser
 from conftest import *
 
 # Marks
 pytestmark = [pytest.mark.linux, pytest.mark.win32, pytest.mark.tier(level=0), pytest.mark.agent]
 
 configurations = load_wazuh_configurations(configurations_path, __name__)
 tests = load_tests(os.path.join(test_data_path, 'wazuh_enrollment_tests.yaml'))
-AGENT_AUTH_TIMEOUT = 5
+host_name = socket.gethostname()
+MANAGER_ADDRESS = '127.0.0.1'
+AGENT_AUTH_TIMEOUT = 20
```

Comment:
In my local runs, both the `log_monitor` as well as the `socket_listener` finalize the execution as soon as they find the log or event that they are looking for.
I decided to increase the time just in case any of the monitors could face a delay and also be aligned with the other tests.